### PR TITLE
Fix incorrect dissociation array destructuring

### PR DESCRIPTION
**Pull request by janklan**
Fixes https://github.com/DamienHarper/auditor/issues/91

All changes outside of src/Provider/Doctrine/Auditing/Transaction/TransactionProcessor.php are a result of running `composer csfixer`. Sorry for the clutter. 

FYI the test suite did not detect this problem, even though pcov shows the line covered. I didn't look into the reasons of that.

### CODE REVIEW

File: src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php [hunk 1]
```diff
@@ -19,10 +19,8 @@ trait AuditTrait
      * @throws \DH\Auditor\Exception\MappingException
      * @throws \Doctrine\DBAL\Exception
      * @throws \Doctrine\ORM\Mapping\MappingException
-     *
-     * @return mixed
      */
-    private function id(EntityManagerInterface $entityManager, object $entity)
+    private function id(EntityManagerInterface $entityManager, object $entity): mixed
```
Comment (DamienHarper) [inline src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php:23]: This breaks with PHP < 8.0, please remove the return typehint

File: src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php [hunk 2]
```diff
@@ -66,10 +64,8 @@ private function id(EntityManagerInterface $entityManager, object $entity)
      *
      * @throws \Doctrine\DBAL\Exception
      * @throws \Doctrine\DBAL\Types\ConversionException
-     *
-     * @return mixed
      */
-    private function value(EntityManagerInterface $entityManager, Type $type, $value)
+    private function value(EntityManagerInterface $entityManager, Type $type, $value): mixed
```
Comment (DamienHarper) [inline src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php:68]: This breaks with PHP < 8.0, please remove the return typehint

File: src/Provider/Doctrine/Persistence/Reader/Filter/RangeFilter.php [hunk 3]
```diff
@@ -36,18 +36,12 @@ public function getName(): string
         return $this->name;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getMinValue()
+    public function getMinValue(): mixed
```
Comment (DamienHarper) [inline src/Provider/Doctrine/Persistence/Reader/Filter/RangeFilter.php:39]: This breaks with PHP < 8.0, please remove the return typehint

File: src/Provider/Doctrine/Persistence/Reader/Filter/RangeFilter.php [hunk 4]
```diff
@@ -36,18 +36,12 @@ public function getName(): string
...
     {
         return $this->minValue;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getMaxValue()
+    public function getMaxValue(): mixed
```
Comment (DamienHarper) [inline src/Provider/Doctrine/Persistence/Reader/Filter/RangeFilter.php:44]: This breaks with PHP < 8.0, please remove the return typehint

**Review by DamienHarper — CHANGES_REQUESTED** [review]
Thanks @janklan for the contribution! There are a few changes that need to be rolled back because of PHP < 8.0 compatibility. Once they're fixed, I will merge this PR.

**Comment by janklan** [comment]
I removed the commit where I ran the PHP CS Fixer, which is the reason why there were return typehints. Let's see if it now passes. I tohught PHP CS fixer respects the version constraints imposed by Composer; I learn something new every day :)

**Review by DamienHarper — APPROVED** [review]

**Pull request merged**